topics/fediverse/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: fedi
```

Comment:
```suggestion
```

I only see [one public repository using the `fedi` topic](https://github.com/topics/fedi), so I'm not sure it's a helpful alias.

Reply:
Hmm, fair point.

I'll apply this suggestion because it actually seems most repository owners are descriptive about the topics they add 😄